### PR TITLE
Game.edits

### PR DESCRIPTION
**Pull request by aparker22**

### CODE REVIEW

**Comment by aparker22** [comment]
@SagePadawan

File: game2.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+
+
+#  this is a timer for the badguys
+badtimer=100
+badtimer1=0
+badguys=[[640,100]]
+bad_list = []
+
+
```
Comment (SagePadawan) [inline game2.py:22]: #Really well commented code!

File: game2.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+#  Game Logic
+while 1:
+    badtimer -= 1
+    screen.blit(court, (0, 0))
+    screen.blit(goal,(487, 0))
+    
+    #This sets the initital positions
+    position = pygame.mouse.get_pos()
+    angle = math.atan2(position[1]-(playerpos[1]+32),position[0]-(playerpos[0]+26))
+    playerrot = pygame.transform.rotate(player, 360-angle*57.29)
```
Comment (SagePadawan) [inline game2.py]: Adding quick comments to the individual lines may help readability. e.g. I'm guessing the "playerrot" var sets the rotation, but I'm not sure.

Comment (aparker22) [reply game2.py]: yes it does!
Thank you!

**Pull request merged**